tesseract_common/include/tesseract_common/serialization.h

```diff
@@ -48,82 +45,38 @@ namespace serialization
 /****** Eigen::VectorXd ******/
 /*****************************/
 template <class Archive>
-inline void save(Archive& ar, const Eigen::VectorXd& g, const unsigned int /*version*/)
-{
-  long rows = g.rows();
-  ar& BOOST_SERIALIZATION_NVP(rows);
-  ar& boost::serialization::make_nvp("data", boost::serialization::make_array(g.data(), rows));
-}
+void save(Archive& ar, const Eigen::VectorXd& g, const unsigned int version);  // NOLINT
```

Comment:
It might be nice at some point (maybe not in the scope of this PR) to try to serialize the `Matrix` or `MatrixBase` class to be able to serialize vectors and matrices of any shape/number type using additional template parameters to specify these dimensions/types. I tried doing this quickly the other day, but wasn't all that successful. Something to think about for the future

Reply:
I agree, I had looked at this a while back and found examples but was unsure if this would cause issues with the custom Isometry3d. I will take another look in the near future.

Reply:
`Isometry3d` is a typedef of `Transform`, so I wouldn't think it would cause issues

Reply:
That is good. I will see if I can find the code again and add it.